File: lista1_js/8.div_soma_dif.js
//8. Leia 2 números, calcule e escreva a divisão da soma pela subtração dos números lidos.
import { question } from "readline-sync"

//Entrada
const num1 = Number(question('Digite o 1º número: '))
const num2 = Number(question('Digite o 2º número: '))

//Processamento
const soma = num1 + num2
const diferenca = num1 - num2
const divisao = soma / diferenca

//Saída
console.log(`\n>>> A divisão da soma: ${soma} pela subtração: ${diferenca} é ${divisao.toFixed(1)}.`)

File: lista1_py/8.div_soma_dif.py
#8. Leia 2 números, calcule e escreva a divisão da soma pela subtração dos números lidos.

#Entrada
num1 = float(input('Digite o 1º número: '))
num2 = float(input('Digite o 2º número: '))

#Processamento
soma = num1 + num2
diferenca = num1 - num2
divisao = soma / diferenca

#Saída
print(f'\n>>> A divisão da soma: {soma} pela subtração: {diferenca} é {divisao:.1f}.')